app/helpers/sheet/access_grant.rb

```diff
@@ -0,0 +1,10 @@
+#  Copyright (c) 2020, Pfadibewegung Schweiz. This file is part of
```

Comment:
Diese Datei könnte man noch nach app/helpers/sheet/oauth/access_grant.rb verschieben oder?

Reply:
Hm, ehrlich gesagt verstehe ich die Sheet-Hierarchie nicht komplett 😓 
`app/helpers/sheet/oauth/access_grant.rb` existiert allerdings bereits: In diesem Ordner sind die Sheets für die Tabs der «OAuth Applikationen»-Einstellungen.